CHANGELOG.md

```diff
@@ -52,3 +52,4 @@ Please add a _short_ line describing the PR you make, if the PR implements a spe
 - Make previous HOTP invalid at password reset ([#1054](https://github.com/ScilifelabDataCentre/dds_web/pull/1054))
 - New endpoint for listing Units as Super Admin ([1060](https://github.com/ScilifelabDataCentre/dds_web/pull/1060))
 - New endpoint for listing unit users as Super Admin ([#1059](https://github.com/ScilifelabDataCentre/dds_web/pull/1059))
+- Return int instead of string from files listing and only return usage info if right role ()
```

Comment:
```suggestion
- Return int instead of string from files listing and only return usage info if right role ([#1070](https://github.com/ScilifelabDataCentre/dds_web/pull/1070))
```